paint-timing/fcp-only/fcp-bg-image-set.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP due to background image in image-set</title>
+<style>
+    #main {
+        width: 100px;
+        height: 100px;
+    }
+
+    .contentful {
+        background-image: -webkit-image-set(url(../resources/circles.png) 1x);
```

Comment:
Should this also have image-set with the same, to make it more future-proof?

Reply:
ok